### DISCUSSION TITLE
How to show sponsor button below issues?

### DISCUSSION THREAD

**Discussion by wojciech-kulik** [community/community#114429] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I noticed that in some repositories the sponsor button appears below issues. 
Like here for example: https://github.com/jesseduffield/lazygit/issues/3438

In my repository, it doesn't work ([example](https://github.com/wojciech-kulik/xcodebuild.nvim/issues/87)). How can I enable it? 

Please note, that GitHub sponsors is configured and enabled for my repository and it appears on the main page, in the section on the right side.

**Comment by celestinod** [comment, marked as answer]
Hey Woj!

For me it is being shown at the bottom of the page.

![image](https://github.com/community/community/assets/164261793/aee5c64b-d1eb-4ff0-896b-33dfa079634b)

I believe it doesn't appears to you because you are the owner of the repo and most likely won't be sponsoring your own repo.

Hope you find this helpfull!

**Reply by wojciech-kulik** [reply]
Oh, that's interesting. I was suspecting something like that and I even checked it from the incognito mode, but with no luck. Apparently, you have to be also logged in from another account :D. Thank you for verifying it!
